templates/mail/htmlmail.php

```diff
@@ -11,7 +11,8 @@
 <tr>
 <td width="20px">&nbsp;</td>
 <td style="font-weight:normal; font-size:0.8em; line-height:1.2em; font-family:verdana,'arial',sans;">
-Hello,<br>
+<?php print_unescaped($l->t('Hello,')); ?>
```

Comment:
Hmmm - maybe just `Hello` should be fed to the translator method. Or maybe it is good that the string `Hello,` will need to have a translation available?

Reply:
We can add a note to translators which will popup in transifex to let them know.

Reply:
From my understanding, `Hello,` will be added to Transifex at the next sync. The translation can then be added without the comma in the individual languages anyway?